CHANGELOG.md

```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v8.0.0](https://github.com/voxpupuli/puppet-kibana/tree/v8.0.0) (2023-08-04)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-kibana/compare/v7.0.1...v8.0.0)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#74](https://github.com/voxpupuli/puppet-kibana/pull/74) ([bastelfreak](https://github.com/bastelfreak))
+
+**Implemented enhancements:**
+
+- Support alternative package & service name for Kibana [\#16](https://github.com/voxpupuli/puppet-kibana/issues/16)
+- Add Puppet 8 support [\#76](https://github.com/voxpupuli/puppet-kibana/pull/76) ([bastelfreak](https://github.com/bastelfreak))
+- Remove template erb and puppet\_x folder [\#72](https://github.com/voxpupuli/puppet-kibana/pull/72) ([phaedriel](https://github.com/phaedriel))
+- Add optional parameter plugindir \(Directory containing kibana plugins\) [\#71](https://github.com/voxpupuli/puppet-kibana/pull/71) ([phaedriel](https://github.com/phaedriel))
+- Add sensitive for kibana config [\#68](https://github.com/voxpupuli/puppet-kibana/pull/68) ([phaedriel](https://github.com/phaedriel))
+
+**Closed issues:**
+
+- kibana plugin installation and directory permissions issue [\#69](https://github.com/voxpupuli/puppet-kibana/issues/69)
+- bundle exec rake spec failed [\#62](https://github.com/voxpupuli/puppet-kibana/issues/62)
```

Comment:
I added the `skip-changelog` label to both. please regenerate the changelog, then both issues should disappear.

Reply:
Ah, I removed "enhancement" from #16 so now it appears here.  Do you want to skip-changelog on it too?

Reply:
I think it's fine now.

Reply:
okay, if we are fine here (?) i'm going to merge it